sources/packages/backend/libs/sims-db/src/entities/disbursement-overaward.model.ts

```diff
@@ -97,4 +100,31 @@ export class DisbursementOveraward extends RecordDataModel {
     nullable: true,
   })
   deletedAt?: Date;
+  /**
+   * Note entered during manual overaward record added.
+   */
+  @OneToOne(() => Note, { eager: false, cascade: true })
+  @JoinColumn({
+    name: "note_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  overawardNotes: Note;
+
+  /**
+   * User who added overaward manual record.
+   */
+  @OneToOne(() => User, { eager: true, cascade: true })
```

Comment:
I believe that we had some back and forth along the time, but I believe that we should use `cascade: false` for these ones and we should defiantly use `eager: false`.